### PR TITLE
fix(dashmate): failing without DOCKER_COMPOSE_V2 env

### PR DESCRIPTION
**Pull request by pshenmic**
## Issue being fixed or feature implemented

Remove checks for legacy docker-compose commands, as we only support docker compose v2 now.

## What was done?

Removed the check for DOCKER_COMPOSE_V2 flag and `docker-compose` command presence.

## How Has This Been Tested?

Locally

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by strophy** [comment]
Let's specify `DockerCompose.DOCKER_COMPOSE_MIN_VERSION = '2.0.0'` on line 418?

**Review by shumkov — CHANGES_REQUESTED** [review]
There is one more in DockerCompose#up

File: packages/dashmate/src/docker/DockerCompose.js [hunk 1]
```diff
@@ -109,10 +109,9 @@ class DockerCompose {
     await this.throwErrorIfNotInstalled();
 
     try {
-      await execAsync(
-        'docker compose up --no-build -d',
-        this.getOptions(envs),
-      );
+      await dockerCompose.upAll({
```
Comment (shumkov) [inline packages/dashmate/src/docker/DockerCompose.js:112]: We probably need to pass `no-build` flag there?

**Comment by pshenmic** [comment]
Closing in favor of https://github.com/dashevo/platform/pull/441

**Pull request closed without merging**